lib/docusign_rest/client.rb

```diff
@@ -1596,6 +1596,23 @@ def add_envelope_recipients(options={})
       JSON.parse(response.body)
     end
 
+    def update_notification_settings(envelope_id)
+      content_type = {'Content-Type' => 'application/json'}
+      uri = build_uri("/accounts/#{@acct_id}/envelopes/#{envelope_id}/notification")
+      post_body = {
+        expirations: {
+          expireEnabled: 'true',
+          expireAfter: 30
```

Comment:
let's put day value in env config on manager. here you can check if manager has send any value set that value else set default to 120

Reply:
@pramodtech ok